app/Models/Article.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Article extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['title', 'text', 'image', 'category_id'];
+
+    protected $with = ['category', 'tags'];
```

Comment:
Do really need to eager load categories and tags every time? Be very careful with this. For example in your manage/articles you don't use them

Reply:
Thank you i will keep this in mind.